client/components/community-translator/index.jsx

```diff
@@ -31,6 +31,7 @@ class CommunityTranslator extends Component {
 	initialized = false;
 
 	componentDidMount() {
+		this.props.fetchUserSettings();
```

Comment:
Or we could use the query component?

Reply:
Yup! Added via https://github.com/Automattic/wp-calypso/pull/50027/commits/4c6deef2d518ba18206f65560e0878471bc90b08 over in https://github.com/Automattic/wp-calypso/pull/50027